renamings_history.csv

```diff
@@ -0,0 +1 @@
+OLD_NAME,NEW NAME,TYPE
```

Comment:
```suggestion
OLD NAME,NEW NAME,TYPE
```

Reply:
I have not noticed this before, but this seems to be a bug in gap. When the csv file is auto generated, `OLD_NAME` is written instead of `OLD NAME`. I will look into this.
To test this, type in the following commands in gap:

```
R := [rec(OLD_NAME := "old", NEW_NAME := "new", TYPE := "func")];
PrintCSV("renamings_history.csv", R, ["OLD_NAME", "NEW_NAME", "TYPE"]);
```

Reply:
Yup, looks like a bug. Printing `OLD_NAME` is correct though. Instead its printing `NEW NAME` when it should be printing `NEW_NAME`. Nice catch. :)

Did you already report this?